streams/src/main/java/org/apache/kafka/streams/processor/internals/TopologyMetadata.java

```diff
@@ -447,11 +457,106 @@ public String getStoreForChangelogTopic(final String topicName) {
     }
 
     void addSubscribedTopicsFromMetadata(final Set<String> topics, final String logPrefix) {
-        applyToEachBuilder(b -> b.addSubscribedTopicsFromMetadata(topics, logPrefix));
```

Comment:
Can we remove the `addSubscribedTopicsFromMetadata` in `InternalTopologyBuilder` then?